newrelic/core/stats_engine.py

```diff
@@ -732,8 +732,13 @@ def notice_error(self, error=None, attributes={}, expected=None, ignore=None, st
             self.__transaction_errors.append(error_details)
 
         # Regardless of whether we record the trace or the event we still
-        # want to increment the metric Errors/all
-        self.record_time_metric(TimeMetric(name='Errors/all', scope='',
+        # want to increment the metric Errors/all unless the error was marked
+        # as expected
+        if self._error_event(error_details)[0].get("error.expected"):
```

Comment:
We already have `is_expected` available in this function, we can just reduce this to `if is_expected:`

Reply:
Ah good call. I had written this before that functionality was added to notice_error